src/main/java/org/commonwl/view/workflow/QueuedWorkflowRepositoryImpl.java

```diff
@@ -19,12 +19,19 @@ public class QueuedWorkflowRepositoryImpl implements QueuedWorkflowRepositoryCus
 
   @Override
   public QueuedWorkflow findByRetrievedFrom(GitDetails retrievedFrom) {
+    // N.B. The migration from Spring Boot 2 to 3 got blocked for a while as
+    //      we couldn't figure out a way to get the mapping of types to work.
+    //      We always ended up the error about the invalid operator for the
+    //      jsonb = bytea types. Found this comment from the author of the
+    //      mapping library we use, with what was the fix for our issue (ran
+    //      out of ideas, so started testing everything found online):
+    //      Use `new JsonType(GitDetails.class)`. That finally solved it.
+    //      Ref: https://github.com/common-workflow-language/cwlviewer/pull/568
```

Comment:
The fix, finally :point_up: 

Works on my computer™, but pending tests with some workflows to confirm it's working.